database/schema.sql

```diff
@@ -36,10 +36,11 @@ CREATE TABLE hearing (
     hearing_id BIGSERIAL PRIMARY KEY,
     judgement_id BIGINT REFERENCES judgement (judgement_id),
     court_id BIGINT REFERENCES court (court_id),
-    case_number VARCHAR(20) NOT NULL,
+    hearing_citation VARCHAR(20) NOT NULL,
```

Comment:
Could you please update the char limit of these to about 50? I just scrolled through the Case Law Website and alot of these citations are longer than 20


```suggestion
    hearing_citation VARCHAR(50) NOT NULL,
```